base/mpfr.jl

```diff
@@ -571,9 +572,8 @@ end
 
 for f in (:log, :log2, :log10)
     @eval function $f(x::BigFloat)
-        if x < 0
-            throw(DomainError(x, string($f, " will only return a complex result if called ",
-                              "with a complex argument. Try ", $f, "(complex(x)).")))
+        if x < 0 && throw(DomainError(x, string($f, " will only return a complex result if called ",
```

Comment:
This is how it's styled on https://github.com/musm/julia/blob/1e86c79d537e80af750d436190d4d56db7e53504/base/mpfr.jl#L586